src/Card.java

```diff
@@ -0,0 +1,49 @@
+public class Card {
+
+    private String faceName, suit;
+    private int faceValue;
+
+    /**
+     * Constructor for card class (learned how to do this on youtube)
+     * @param suit "spades", "clubs", "diamonds", "hearts"
+     * @param face 2, 3, 4, 5, ..., King, Ace
+     * @param value 2, 3, 4, 5, ..., 13, 14
+     */
+    public Card(String suit, String face, int value) {
+
+        this.suit = suit;   //I have two "suit" so this.suit refers to the suit outside
+        faceName = face;
+        faceValue = value;
+
+    }
+
+    /**
+     * This returns the face and suit of the Card object as a string
+     * @return
+     */
+    public String toString() {
+
+        return faceName + " of " + suit;
```

Comment:
I recommended Not to use "+" with String. It is best practice to use String.concat() method.
I would write:
`return faceName.concat(" + ").concat(suit);`